conf/machine/include/imx8mn-evk.inc

```diff
@@ -0,0 +1,52 @@
+MACHINEOVERRIDES =. "mx8:mx8m:mx8mn:"
+
+require conf/machine/include/imx-base.inc
+require conf/machine/include/tune-cortexa53.inc
+
+MACHINE_FEATURES += "wifi bluetooth optee jailhouse bcm43455 bcm4356"
```

Comment:
Is `jailhouse` also applicable to mainline BSP? Or is it NXP-specific only?

In later case - it should be appended under `use-nxp-bsp` to `MACHINE_FEATURES` below.

Reply:
Oops, I actually missed looking at this with that last push. I haven't tried it with mainline, so I guess I should move it to NXP BSP.

Reply:
Done.